src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -3890,6 +3891,9 @@ public sealed partial class TimeZoneInfo : System.IEquatable<System.TimeZoneInfo
         void System.Runtime.Serialization.ISerializable.GetObjectData(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
         public string ToSerializedString() { throw null; }
         public override string ToString() { throw null; }
+        public static bool TryConvertIanaIdToWindowsId(string ianaId, out string windowsId) { throw null; }
```

Comment:
Why isn't `out string windowsId` nullable?

Should it be `out string? windowsId` to match `TryConvertWindowsIdToIanaId`? It also looks like this doesn't match the src file.

Reply:
Yes, I just missed it. thanks.

Reply:
I have fixed that.